events/management/commands/remove_unused_permissions.py

```diff
@@ -0,0 +1,92 @@
+from dateutil.relativedelta import relativedelta
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.db.models.functions import Greatest
+from django.utils import timezone
+
+from helevents.models import User
+from registrations.models import RegistrationUserAccess
+
+
+class Command(BaseCommand):
+    help = "Remove unused admin permissions after expiration time has passed."
+
+    @staticmethod
+    def _handle_event_admins(datetime_now):
+        admins_count = 0
+
+        threshold_time = datetime_now - relativedelta(
+            months=settings.EVENT_ADMIN_EXPIRATION_MONTHS
+        )
+        for user in User.objects.filter(
+            last_login__lt=threshold_time, admin_organizations__isnull=False
+        ):
+            user.admin_organizations.clear()
+            admins_count += 1
+
+        return admins_count
+
+    @staticmethod
+    def _handle_financial_admins(datetime_now):
+        admins_count = 0
+
+        threshold_time = datetime_now - relativedelta(
+            months=settings.FINANCIAL_ADMIN_EXPIRATION_MONTHS
+        )
+        for user in User.objects.filter(
+            last_login__lt=threshold_time, financial_admin_organizations__isnull=False
+        ):
+            user.financial_admin_organizations.clear()
+            admins_count += 1
+
+        return admins_count
+
+    @staticmethod
+    def _handle_registration_admins(datetime_now):
+        admins_count = 0
+
+        threshold_time = datetime_now - relativedelta(
+            months=settings.REGISTRATION_ADMIN_EXPIRATION_MONTHS
+        )
+        for user in User.objects.filter(
+            last_login__lt=threshold_time,
+            registration_admin_organizations__isnull=False,
+        ):
+            user.registration_admin_organizations.clear()
+            admins_count += 1
+
+        return admins_count
+
+    @staticmethod
+    def _handle_registration_users(datetime_now):
+        admins_count = 0
```

Comment:
This could be renamed since these are not admins.

Reply:
Changed as suggested. Also mentioned registration users separately in the command's help text.